ui/job-view/pushes/PushHeader.jsx

```diff
@@ -66,14 +66,17 @@ function PushCounts(props) {
   const percentComplete = getPercentComplete(props);
 
   return (
-    <span className="push-progress">
-      {percentComplete === 100 && <span>- Complete -</span>}
-      {percentComplete < 100 && total > 0 && (
-        <span title="Proportion of jobs that are complete">
-          {percentComplete}% - {inProgress} in progress
-        </span>
-      )}
-    </span>
+    <div>
+      <span className="fixed-by-commit">FixedByCommit goes here</span>
```

Comment:
Once you have added the ``filteredByCommit`` value to the ``Push`` state, and then passed it into ``PushHeader`` as a prop, then you would put that value here.  But you will want to check if it is greater than 0 to decide whether to show this ``span`` at all.